configuration/forms.md

```diff
@@ -46,6 +46,26 @@ Used for SMS interfaces such as formatted SMS, SIM applications, and Medic Colle
   }
 }
 ```
+## JSON Form Schema
+| property | type | description | required |
+|---|---|---|---|
+| `meta` | object | Information about the report | yes |
+| `meta.code` | string | The unique form identifier, which will be sent with all reports of this form | yes |
+| `meta.icon` | string | Name of the icon resource shown in the app. | no |
+| `meta.translation_key` | string | Name of the form shown in the app. | no |
+| `fields`| object | Collection of field objects included in the form. | yes |
+| `fields.${field}` | object | Field that is part of the form. | yes |
+| `fields.${field}.type` | string | Data type of field. See [supported data types](#supported-data-types). | yes |
+| `fields.${field}.labels` | object | | no |
+| `fields.${field}.labels.short` | string, object with `translation_key` field, or translation object | Label shown for field in the app, seen when report is viewed in Reports tab. If missing, label will default to a translation key of `report.${form_name}.${field_name}` (eg `report.f.patient_id`) which can be translated in the app languages. | no |
+| `fields.${field}.labels.tiny` | string | Unique identifier within the form for this field. Used in form submission to bind values to fields. Not required for all submission formats. | no |
+| `fields.${field}.position` | integer | Zero based order of this field for incoming reports. | no |
+| `fields.${field}.flags` | object | Additional instructions that could be used by form renderers. For instance `{ "input_digits_only": true }` indicated to SIM applications to show the number keyboard. Obsolete. | no |
+| `fields.${field}.length` | array with two integers | Inclusive range accepted for length of the field. | no |
+| `fields.${field}.required` | boolean | Determines if a report without this field is considered valid. | no |
+| `public_form` | boolean | Determines if reports will be accepted from phone numbers not associated to a contact. Set to false if you want to reject reports from unknown senders. Default: true | no |
```

Comment:
Up until 3.4 (inclusive), only `registration` transition used `public_form`.
In 3.5 (via this issue https://github.com/medic/medic/issues/5365 and this branch https://github.com/medic/medic/tree/5365-phone-verification)  I am updating the following transitions to make use of `public_form` as well:
- accept_patient_reports
- conditional_alerts
- death_reporting
- multi_report_alerts
- muting
- update_notifications

(I chose the transitions which can produce changes in other docs besides the one they run on). We can discuss if you'd like more (or less) to be included in this list.
Nevertheless, this is a feature that is not yet available for all transitions.

Reply:
Thanks for the update, sounds like we should document that accordingly for 3.5 release.